docs/pages/includes/plugins/rbac.mdx

```diff
@@ -27,8 +18,75 @@ spec:
         verbs: ['list', 'read', 'update']
       - resources: ['access_plugin_data']
         verbs: ['update']
+---
+kind: user
+metadata:
+  name: access-plugin
+spec:
+  roles: ['access-plugin']
+version: v2
+```
+
+Create the user and role:
+
+```code
+$ tctl create -f access-plugin.yaml
+```
+
+As with all Teleport users, the Teleport Auth Service authenticates the
+`access-plugin` user by issuing short-lived TLS credentials. In this case, we
+will need to request the credentials manually by _impersonating_ the
+`access-plugin` role and user.
+
+<ScopedBlock scope={["oss", "enterprise"]}>If you are using `tctl` from the Auth
+Service host, you will already have impersonation privileges.</ScopedBlock>
+
+To grant your user impersonation privileges for `access-plugin`, define a role
+called `access-plugin-impersonator` by pasting the following YAML document into
+a file called `access-plugin-impersonator.yaml`:
+
+```yaml
+kind: role
+version: v4
+metadata:
+  name: access-plugin-impersonator
+spec:
+  allow:
+    impersonate:
+      roles:
+      - access-plugin
+      users:
+      - access-plugin
+```
+
+Create the `access-plugin-impersonator` role: 
+
+```code
+$ tctl create -f access-plugin-impersonator.yaml
+```
+
+Retrieve your user definition:
+
+```code
+$ TELEPORT_USER=$(tsh status | awk -F "      " '/Logged in as/{print $2}')
```

Comment:
Would rather not rely on the text output, wouldn't be surprised if we break that at some point. It looks like we use `jq` elsewhere in the docs so this might be a better option

```suggestion
$ TELEPORT_USER=$(tsh status --format=json | jq -r .active.username)
```